cypress/integration/brewery/CreateScenario-tests.spec.js

```diff
@@ -123,9 +123,14 @@ describe('Create scenario', () => {
       }
     );
 
+    // Check parameters accordion state after creating scenario
+    ScenarioParameters.getParametersTabs().should('be.visible');
```

Comment:
Are you sure `getParametersTabs` returns the parameters accordion ? If it returns the parameters card (that contains the accordion), then it will always be visible and your test does not check the accordion behavior.

Reply:
In fact    ` ScenarioParameters.getParametersTabs().should('be.visible');` allow to check if the accordion is correctly expanded.

Reply:
Ok my bad, I thought it returned the whole parameters card, but it returns the container of the tabs instead :+1: